packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -152,7 +152,7 @@ export const SetupFlopFlipProvider = (props: TSetupFlopFlipProviderProps) => {
 
   const defaultFlags = useMemo(
     () => ({
-      ...featureFlags.FLAGS,
+      ...featureFlags.DEFAULT_FLAGS,
```

Comment:
This is the main fix. We only want to pass default flags here, not the short-lived flags.